[2024-07-30T10:26:33.555+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:26:33.583+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-02T00:00:00+00:00 [queued]>
[2024-07-30T10:26:33.594+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-02T00:00:00+00:00 [queued]>
[2024-07-30T10:26:33.594+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:26:33.736+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-02 00:00:00+00:00
[2024-07-30T10:26:33.746+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=928) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:26:33.749+0000] {standard_task_runner.py:63} INFO - Started process 931 to run task
[2024-07-30T10:26:33.746+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-02T00:00:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmplxe6uymr']
[2024-07-30T10:26:33.751+0000] {standard_task_runner.py:91} INFO - Job 87: Subtask Run_data_quality_checks
[2024-07-30T10:26:33.993+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-02T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:26:34.388+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-02T00:00:00+00:00'
[2024-07-30T10:26:34.389+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:26:34.407+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:26:34.427+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:34.449+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:26:34.453+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:34.456+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 266 records
[2024-07-30T10:26:34.457+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:26:34.469+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:34.491+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:26:34.496+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:34.500+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 41 records
[2024-07-30T10:26:34.501+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:26:34.515+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:34.536+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:26:34.542+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:34.544+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:26:34.546+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:26:34.560+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:34.578+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:26:34.584+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:34.586+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:26:34.587+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:26:34.600+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:26:34.617+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:26:34.621+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:26:34.623+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 266 records
[2024-07-30T10:26:34.624+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:26:34.640+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-02T00:00:00+00:00, execution_date=20181102T000000, start_date=20240730T102633, end_date=20240730T102634
[2024-07-30T10:26:34.728+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:26:34.750+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
